integration/base/src/main/java/com/inrupt/client/integration/base/AccessGrantScenarios.java

```diff
@@ -154,17 +152,19 @@ static void setup() throws IOException {
             podUrl += Utils.FOLDER_SEPARATOR;
         }
 
-
         privateContainerURI = URIBuilder.newBuilder(URI.create(podUrl))
                 .path(State.PRIVATE_RESOURCE_PATH + "/").build();
 
+        authResourceOwnerClient = createAuthenticatedClient();
+        //if a tests fails it can be that the cleanup was not properly done, so we do it here too
+        Utils.deleteContentsRecursively(authResourceOwnerClient, privateContainerURI);
```

Comment:
We can't do this deletion here, because it will cause any other concurrently-running test to fail.